src/init/features/storage.ts

```diff
@@ -1,17 +1,15 @@
-"use strict";
+import * as clc from "cli-color";
+import * as fs from "fs";
 
-var clc = require("cli-color");
-var fs = require("fs");
+import * as logger from "../../logger";
+import { prompt } from "../../prompt";
 
-var { prompt } = require("../../prompt");
-var logger = require("../../logger");
-
-var RULES_TEMPLATE = fs.readFileSync(
+const RULES_TEMPLATE = fs.readFileSync(
   __dirname + "/../../../templates/init/storage/storage.rules",
   "utf8"
 );
 
-module.exports = function(setup, config) {
+export async function doSetup(setup: any, config: any): Promise<any> {
```

Comment:
Can this be `Promise<unknown>` or `Promise<void>`? I don't think the return value of this promise is ever used, right? It may be worth removing the `return` of `config.writeProjectFile` and letting this signature be `Promise<void>`.

Reply:
Done